packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx

```diff
@@ -139,12 +145,27 @@ export const Docs: () => JSX.Element = () => {
         CallComposite. Check out [Customize Call
         Controls](./?path=/docs/composites-call-basicexample--basic-example#customize-call-controls) to learn more.
       </Description>
-
+      <div ref={refImageSharing}>
+        <Heading>Adding image sharing</Heading>
+      </div>
+      <div ref={refImageSharingTeams}>
+        <Subheading>Inline Image In Teams Interop Meeting Chat Thread</Subheading>
+        <SingleLineBetaBanner />
+        <Description>
+          The Azure Communication Services end user can recieve images shared by the Teams user without any additional
+          setup when using the `CallWithChat` Composite in a Teams interop scenario,. Azure Communication Services end
+          user would need to join the Teams meeting first, as soon as the Teams user sends a file from the Teams client,
+          the Azure Communication Services end user will be see the image in the chat thread. Please check out the
+          tutorial for [Enable inline image using UI Library in Teams Interoperability
+          Chat](https://learn.microsoft.com/en-us/azure/communication-services/tutorials/inline-image-tutorial-interop-chat)
+          for details.
+        </Description>
+      </div>
       <div ref={refFileSharing}>
         <Heading>Adding file sharing</Heading>
       </div>
       <div ref={refFileSharingACS}>
-        <Subheading>In Azure Communication Services Chat Thread</Subheading>
+        <Subheading>File Sharing In Azure Communication Services Chat Thread</Subheading>
```

Comment:
The original text seems better; no need to repeat the file sharing with the header right above?

Reply:
i agree, but unfortunately, this subheading would be identical anchor tag as inline image causing the anchor not working properly. for example: #In-Azure Communication-Services-Chat-Thread would navigate to inline image section now and will not navigate to file sharing

Reply:
Let's revisit for v7